src/main/java/org/datavyu/plugins/ffmpeg/FfmpegSdlMediaPlayer.java

```diff
@@ -14,33 +15,14 @@
  */
 public final class FfmpegSdlMediaPlayer extends FfmpegMediaPlayer {
 
+    private static final Logger LOGGER = LogManager.getFormatterLogger(MpvMediaPlayer.class);
+
     static {
         try {
-            if(! NativeLibraryLoader.isMacOs) {
-                System.out.println("Extracting Windows libraries for ffmpeg.");
-                NativeLibraryLoader.extract("avutil-56");
-                NativeLibraryLoader.extract("swscale-5");
-                NativeLibraryLoader.extract("swresample-3");
-                NativeLibraryLoader.extract("avcodec-58");
-                NativeLibraryLoader.extract("avformat-58");
-                NativeLibraryLoader.extract("avfilter-7");
-                NativeLibraryLoader.extract("avdevice-58");
-                NativeLibraryLoader.extract("postproc-55");
-                NativeLibraryLoader.extract("SDL2");
-            } else {
-                System.out.println("Extracting Mac OS libraries for ffmpeg.");
-                NativeLibraryLoader.extract("avutil.56");
-                NativeLibraryLoader.extract("swscale.5");
-                NativeLibraryLoader.extract("swresample.3");
-                NativeLibraryLoader.extract("avcodec.58");
-                NativeLibraryLoader.extract("avformat.58");
-                NativeLibraryLoader.extract("avfilter.7");
-                NativeLibraryLoader.extract("avdevice.58");
-                NativeLibraryLoader.extract("postproc.55");
-            }
+            NativeLibraryLoader.extract(FFMPEG_DEPENDENCIES);
```

Comment:
SDL2 Dependency is missing

Reply:
Added + Exception when trying to load on Mac OS since we don't have it working in that case.